### DISCUSSION TITLE
如何在没有wifi环境的情况下使用无线调试？

### DISCUSSION THREAD

**Discussion by achieve-dream1221** [RikkaApps/Shizuku#552] [Q&A]
记得之前，OPPO能够在没有wifi的情况下，直接打开wifi图标，然后通过连接127.0.0.1:5555进行连接，在更新系统后，必须要连接到wifi才能使用无线调试功能，如果外出不方便连接wif，那么就无法启动shizuku，依赖于shizuku的应用就不能正常使用，导致冻结的应用不能打开。
![cbdd4a0242e23dd9841fad8c125b738](https://github.com/user-attachments/assets/eacb7b22-db8f-4fe5-a21c-d671041ed704)

**Comment by GeekerKuan** [comment]
![Screenshot_20240910-212021.png](https://github.com/user-attachments/assets/adfdcdae-66e8-4881-b120-99e05a6e9a04)

可以试一下 雹APP 的“shizuku-暂停”模式来实现应用的冻结![Screenshot_20240910-212250.png](https://github.com/user-attachments/assets/e352e7e4-f525-4492-beaa-5767839f4550)

冻结后的APP在启动器仍会显示，不过为灰色
![Screenshot_20240910-212253.png](https://github.com/user-attachments/assets/ee1a47c1-9be1-4d8b-ba9b-6ff9ffe2fcd7)

点开APP图标后在弹窗点击取消暂停应用即可取消冻结(无需shizuku)
